### PR TITLE
AC-667: Create phpcs static check for EmailTemplateTest

### PR DESCRIPTION
**Pull request by jcuerdo**

### CODE REVIEW

File: Magento2/Tests/Legacy/EmailTemplateUnitTest.2.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<h1>{{htmlescape "H1"}}</h1>
+<h2>{{escapehtml "H2"}}</h2>
+<p>{{escapehtml "p"}} {{htmlescape "p"}}</p>
+<p class="greeting">{{trans "Translateme"}}</p>
```
Comment (loginesta) [inline Magento2/Tests/Legacy/EmailTemplateUnitTest.2.html:4]: no new line at the end of file

**Review by loginesta — DISMISSED** [review]
LGTM

File: Magento2/Tests/Legacy/EmailTemplateUnitTest.2.html (hunk 1 above)
Comment (jcuerdo) [reply Magento2/Tests/Legacy/EmailTemplateUnitTest.2.html]: Done

File: Magento2/ruleset.xml [hunk 2]
```diff
@@ -242,6 +242,13 @@
         <type>warning</type>
     </rule>
```
Comment (sivaschenko) [inline Magento2/ruleset.xml:244]: I'd remove this empty line for consistency.

File: Magento2/ruleset.xml [hunk 3]
```diff
@@ -242,6 +242,13 @@
...
 
+    <rule ref="Magento2.Legacy.EmailTemplate.FoundObsoleteEmailDirective">
+        <include-pattern>view/email/*.html</include-pattern>
+        <include-pattern>view/*/email/*.html</include-pattern>
+        <severity>8</severity>
+        <type>warning</type>
```
Comment (sivaschenko) [inline Magento2/ruleset.xml:249]: I think this should be an error as using of obsolete directives will actually harm the functionality of email templates

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Great first contribution, @jcuerdo ! Please see my comments

**Review by svera — DISMISSED** [review]

File: Magento2/ruleset.xml [hunk 4]
```diff
@@ -241,6 +241,12 @@
         <severity>8</severity>
         <type>warning</type>
     </rule>
+    <rule ref="Magento2.Legacy.EmailTemplate.FoundObsoleteEmailDirective">
+        <include-pattern>view/email/*.html</include-pattern>
+        <include-pattern>view/*/email/*.html</include-pattern>
+        <severity>8</severity>
```
Comment (sivaschenko) [inline Magento2/ruleset.xml:247]: Please move it to the section with errors in the beginning of this file
```suggestion
        <severity>10</severity>
```

**Review by sivaschenko — APPROVED** [review]

**Comment by jcuerdo** [comment]
@magento import pr to magento-commerce/magento-coding-standard

**Comment by magento-engcom-team** [comment]
@jcuerdo the pull request successfully imported.

**Pull request merged**